build.sbt

```diff
@@ -548,6 +536,25 @@ lazy val `scio-core`: Project = project
     buildInfoPackage := "com.spotify.scio"
   )
 
+lazy val `scio-cosmosdb`: Project = project
+  .in(file("scio-cosmosdb"))
+  .configs(IntegrationTest)
+  .settings(itSettings)
+  .settings(commonSettings)
+  .settings(publishSettings)
+  .dependsOn(`scio-core`, `scio-test` % "test;it")
+  .settings(
+    //scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-Xsource:3"), // , "-Ymacro-annotations"
+    scalacOptions ++= Seq("-Xsource:3"),
```

Comment:
Because the others modules not support the syntax of scala3, 

```scala
sbt:scio> scio-core/compile
[info] compiling 111 Scala sources and 18 Java sources to /home/miuler/proyectos/scio/scio-core/target/scala-2.13/classes ...
[error] /home/miuler/proyectos/scio/scio-core/src/main/scala/com/spotify/scio/ScioContext.scala:203:48: method apply in object PipelineOptionsParser cannot be accessed as a member of object com.spotify.scio.ContextAndArgs.PipelineOptionsParser from <$anon: com.spotify.scio.ContextAndArgs.TypedArgsParser[T,scala.util.Try]>
[error] error after rewriting to ContextAndArgs.this.PipelineOptionsParser.<apply: error>
[error] possible cause: maybe a wrong Dynamic method signature?
[error]         override def parser: ArgsParser[Try] = PipelineOptionsParser[T]()
[error]                                                ^
[error] /home/miuler/proyectos/scio/scio-core/src/main/scala/com/spotify/scio/ScioContext.scala:223:16: method apply in object DefaultParser cannot be accessed as a member of object com.spotify.scio.ContextAndArgs.DefaultParser from object ContextAndArgs in package scio
[error] error after rewriting to ContextAndArgs.this.DefaultParser.<apply: error>
[error] possible cause: maybe a wrong Dynamic method signature?
[error]     withParser(DefaultParser[PipelineOptions]()).apply(args)
[error]                ^
[error] /home/miuler/proyectos/scio/scio-core/src/main/scala/com/spotify/scio/ScioContext.scala:785:48: method apply in object KVCoder cannot be accessed as a member of object com.spotify.scio.coders.KVCoder from class ScioContext in package scio
[error] error after rewriting to com.spotify.scio.coders.KVCoder.<apply: error>
[error] possible cause: maybe a wrong Dynamic method signature?
[error]       val coder = CoderMaterializer.beam(this, KVCoder(koder, voder))
```